vllm/entrypoints/openai/api_server.py

```diff
@@ -182,7 +183,8 @@ async def create_chat_completion(raw_request: Request):
                                      "logit_bias is not currently supported")
 
     prompt = await get_gen_prompt(request)
-    error_check_ret = await check_length(request, prompt, engine)
+    model_config = await engine.get_model_config()
```

Comment:
A dump question: Can we cache `model_config` a head of time and avoid this function call at run time?

Reply:
You are right. Let me fix this.